<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuchungsDetails</class>
 <widget class="QDialog" name="BuchungsDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QDateEdit" name="fromDate"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Id"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="preis"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>von</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>bis</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="toDate"/>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="buchungWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>391</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="flugTab">
    <attribute name="icon">
     <iconset>
      <normaloff>flightIcon.png</normaloff>flightIcon.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Flug</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>von</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="flugFromDest">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nach</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="flugToDest">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="flugAirline">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>airline</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="flugBookingClass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>booking class</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mietwagenTab">
    <attribute name="icon">
     <iconset>
      <normaloff>carIcon.png</normaloff>carIcon.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Mietwagen</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pickup Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mietwagenPickupLocation">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Return Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mietwagenReturnLocation">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mietwagenCompany">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicle Class</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mietwagenVehicleClass">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hotelTab">
    <attribute name="icon">
     <iconset>
      <normaloff>hotelIcon.png</normaloff>hotelIcon.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Hotel</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotelName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Town</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="townName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Room Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hotelRoomType">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="zugTab">
    <attribute name="icon">
     <iconset>
      <normaloff>trainIcon.png</normaloff>trainIcon.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Zug</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>von</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainFromDest">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>abfahrt</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="trainTimeAbfahrt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nach</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainToDest">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ankunft</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="trainTimeAnkunft">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ticket Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainTicketTyp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>uber</string>
     </property>
    </widget>
    <widget class="QListView" name="trainConnecting">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="speichern">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abbrechen">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
